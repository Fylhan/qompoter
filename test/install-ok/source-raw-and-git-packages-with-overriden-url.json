{
	"name": "install-ok/qompoter",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"fylhan/posibrain": "v0.2",
		"cp/fylhan": "v1.0"
	},
	"require-dev": {
		"fylhan/platphorm": "dev-master"
	},
	"repositories": {
		"fylhan/posibrain": "https://github.com",
		"fylhan/platphorm": "https://github.com"
	}
}
