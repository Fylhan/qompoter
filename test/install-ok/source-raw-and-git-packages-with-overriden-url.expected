Qompoter
======== install -> create qompoter.lock

* fylhan/posibrain v0.2
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

* cp/fylhan v1.0
  Downloading sources...
  done

* fylhan/platphorm dev-master
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

* cp/fylhan2 v1.0
  Downloading sources...
  done

* cp/acme-cp v1.0
  Downloading sources...
  done

done
